tools/nl/embeddings/data/curated_input/sdg/sheets_svs.csv

```diff
@@ -1,427 +1,427 @@
-dc/topic/sdg_17.8.1,Proportion of individuals using the Internet,,,sdg 17.8.1;sdg 1781;sdg1781;sdg 17 8 1;sdg Proportion of individuals using the Internet;sustainable development goal Proportion of individuals using the Internet;internet access;people with internet access;people who are online;percentage of online people;internet access by genders;proportion of women online;proportion of men online;men with internet access;women with internet access;population with internet access
-dc/topic/sdg_17.9,"Enhance international support for implementing effective and targeted capacity-building in developing countries to support national plans to implement all the Sustainable Development Goals, including through North-South, South-South and triangular cooperation",,,"sdg 17.9;sdg 179;sdg179;sdg 17 9;sdg Enhance international support for implementing effective and targeted capacity-building in developing countries to support national plans to implement all the Sustainable Development Goals, including through North-South, South-South and triangular cooperation;sustainable development goal Enhance international support for implementing effective and targeted capacity-building in developing countries to support national plans to implement all the Sustainable Development Goals, including through North-South, South-South and triangular cooperation"
-dc/topic/sdg_17.9.1,"Dollar value of financial and technical assistance (including through North-South, South-South and triangular cooperation) committed to developing countries",,,"sdg 17.9.1;sdg 1791;sdg1791;sdg 17 9 1;sdg Dollar value of financial and technical assistance (including through North-South, South-South and triangular cooperation) committed to developing countries;sustainable development goal Dollar value of financial and technical assistance (including through North-South, South-South and triangular cooperation) committed to developing countries"
+dcid,sentence
+dc/topic/sdg,Sustainable Development Goals;sdg;sdg;sdg;sdg;sdg Sustainable Development Goals;sustainable development goal Sustainable Development Goals
+dc/topic/sdg_1,No Poverty;sdg 1;sdg 1;sdg1;sdg 1;sdg No Poverty;sustainable development goal No Poverty;poverty rate;poverty
+dc/topic/sdg_1.1,"By 2030, eradicate extreme poverty for all people everywhere, currently measured as people living on less than $1.25 a day;sdg 1.1;sdg 11;sdg11;sdg 1 1;sdg By 2030, eradicate extreme poverty for all people everywhere, currently measured as people living on less than $1.25 a day;sustainable development goal By 2030, eradicate extreme poverty for all people everywhere, currently measured as people living on less than $1.25 a day"

```

Comment:
a comment on how you did these cleanups?

If you used a command line or something, mention it for future reference.

Reply:
Done